tools/docker/Dockerfile.monorepo

```diff
@@ -75,7 +76,8 @@ RUN chmod +x /usr/local/bin/monorepo-entrypoint.sh
 # Set up locale properly
 RUN apt-get update && apt-get install -y locales \
     && locale-gen en_US.UTF-8 \
-    && update-locale LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8
+    && update-locale LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 \
+    && find /var/ -name '*-old' -delete && rm -rf /var/log/dpkg.log /var/log/alternatives.log /var/log/apt/ ~/.launchpadlib
```

Comment:
Adding the `find` here is good, but AFAIK the intermediate layers created above still have them.